shared/my-sites/themes/theme-options.js

```diff
@@ -64,9 +64,15 @@ export default function getButtonOptions( site, theme, actions, setSelectedTheme
 	}
 };
 
-function rawOptions( site, theme ) {
+function rawOptions( site, theme, isLoggedOut ) {
 	return [
 		{
+			name: 'signup',
+			label: i18n.translate( 'Start a Blog' ),
```

Comment:
_Register with this_ doesn't sound adequate to me. Hm, tricky.  _Choose this design_ could work, I think. That, or _Commence Interwebz_. I'm struggling to find something decent. :)